### ISSUE TITLE
CT scientific notation is not supported

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/ph-scale#233]
```
ph-scale : autofill-fuzz : unbuilt : query-parameters
https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/ph-scale/ph-scale_en.html?continuousTest=%7B%22test%22%3A%5B%22ph-scale%22%2C%22autofill-fuzz%22%2C%22unbuilt%22%2C%22query-parameters%22%5D%2C%22snapshotName%22%3A%22snapshot-1629275039864%22%2C%22timestamp%22%3A1629276764713%7D&brand=phet&ea&fuzz&memoryLimit=1000&autoFill=false
Query: brand=phet&ea&fuzz&memoryLimit=1000&autoFill=false
Uncaught Error: Assertion failed: scientific notation is not supported: 1e-10
Error: Assertion failed: scientific notation is not supported: 1e-10
at window.assertions.assertFunction (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/assert/js/assert.js:25:13)
at Object.numberOfDecimalPlaces (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/dot/js/Utils.js:835:17)
at Object.roundToInterval (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/dot/js/Utils.js:853:13)
at GraphIndicatorDragListener.drag [as _dragListener] (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/ph-scale/js/common/view/graph/GraphIndicatorDragListener.js:64:37)
at GraphIndicatorDragListener.drag (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/scenery/js/listeners/PressListener.js:477:10)
at DragListener._dragAction.Action.parameters.name (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/scenery/js/listeners/DragListener.js:251:36)
at Action.execute (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/axon/js/Action.js:227:18)
at GraphIndicatorDragListener.drag (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/scenery/js/listeners/DragListener.js:407:22)
at GraphIndicatorDragListener.pointerMove (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/scenery/js/listeners/PressListener.js:849:14)
at Input.dispatchToListeners (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/scenery/js/input/Input.js:1885:25)
id: Bayes Chrome
Snapshot from 8/18/2021, 2:23:59 AM

----------------------------------

ph-scale : fuzz : built
https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/ph-scale/build/phet/ph-scale_en_phet.html?continuousTest=%7B%22test%22%3A%5B%22ph-scale%22%2C%22fuzz%22%2C%22built%22%5D%2C%22snapshotName%22%3A%22snapshot-1629275039864%22%2C%22timestamp%22%3A1629277556826%7D&fuzz&memoryLimit=1000
Query: fuzz&memoryLimit=1000
Uncaught TypeError: Cannot read property 'length' of undefined
TypeError: Cannot read property 'length' of undefined
at Object.numberOfDecimalPlaces (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/ph-scale/build/phet/ph-scale_en_phet.html?continuousTest=%7B%22test%22%3A%5B%22ph-scale%22%2C%22fuzz%22%2C%22built%22%5D%2C%22snapshotName%22%3A%22snapshot-1629275039864%22%2C%22timestamp%22%3A1629277556826%7D&fuzz&memoryLimit=1000:860:102229)
at Object.roundToInterval (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/ph-scale/build/phet/ph-scale_en_phet.html?continuousTest=%7B%22test%22%3A%5B%22ph-scale%22%2C%22fuzz%22%2C%22built%22%5D%2C%22snapshotName%22%3A%22snapshot-1629275039864%22%2C%22timestamp%22%3A1629277556826%7D&fuzz&memoryLimit=1000:860:102305)
at vD.drag [as _dragListener] (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/ph-scale/build/phet/ph-scale_en_phet.html?continuousTest=%7B%22test%22%3A%5B%22ph-scale%22%2C%22fuzz%22%2C%22built%22%5D%2C%22snapshotName%22%3A%22snapshot-1629275039864%22%2C%22timestamp%22%3A1629277556826%7D&fuzz&memoryLimit=1000:860:1427831)
at vD.drag (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/ph-scale/build/phet/ph-scale_en_phet.html?continuousTest=%7B%22test%22%3A%5B%22ph-scale%22%2C%22fuzz%22%2C%22built%22%5D%2C%22snapshotName%22%3A%22snapshot-1629275039864%22%2C%22timestamp%22%3A1629277556826%7D&fuzz&memoryLimit=1000:860:542478)
at Sf._dragAction.Re.parameters.name (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/ph-scale/build/phet/ph-scale_en_phet.html?continuousTest=%7B%22test%22%3A%5B%22ph-scale%22%2C%22fuzz%22%2C%22built%22%5D%2C%22snapshotName%22%3A%22snapshot-1629275039864%22%2C%22timestamp%22%3A1629277556826%7D&fuzz&memoryLimit=1000:860:934302)
at De.execute (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/ph-scale/build/phet/ph-scale_en_phet.html?continuousTest=%7B%22test%22%3A%5B%22ph-scale%22%2C%22fuzz%22%2C%22built%22%5D%2C%22snapshotName%22%3A%22snapshot-1629275039864%22%2C%22timestamp%22%3A1629277556826%7D&fuzz&memoryLimit=1000:860:69373)
at vD.drag (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/ph-scale/build/phet/ph-scale_en_phet.html?continuousTest=%7B%22test%22%3A%5B%22ph-scale%22%2C%22fuzz%22%2C%22built%22%5D%2C%22snapshotName%22%3A%22snapshot-1629275039864%22%2C%22timestamp%22%3A1629277556826%7D&fuzz&memoryLimit=1000:860:935335)
at vD.pointerMove (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/ph-scale/build/phet/ph-scale_en_phet.html?continuousTest=%7B%22test%22%3A%5B%22ph-scale%22%2C%22fuzz%22%2C%22built%22%5D%2C%22snapshotName%22%3A%22snapshot-1629275039864%22%2C%22timestamp%22%3A1629277556826%7D&fuzz&memoryLimit=1000:860:544936)
at wd.dispatchToListeners (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/ph-scale/build/phet/ph-scale_en_phet.html?continuousTest=%7B%22test%22%3A%5B%22ph-scale%22%2C%22fuzz%22%2C%22built%22%5D%2C%22snapshotName%22%3A%22snapshot-1629275039864%22%2C%22timestamp%22%3A1629277556826%7D&fuzz&memoryLimit=1000:860:632771)
at wd.dispatchEvent (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/ph-scale/build/phet/ph-scale_en_phet.html?continuousTest=%7B%22test%22%3A%5B%22ph-scale%22%2C%22fuzz%22%2C%22built%22%5D%2C%22snapshotName%22%3A%22snapshot-1629275039864%22%2C%22timestamp%22%3A1629277556826%7D&fuzz&memoryLimit=1000:860:632401)
id: Bayes Chrome
Snapshot from 8/18/2021, 2:23:59 AM

----------------------------------

ph-scale : fuzz : built-phet-io
https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/ph-scale/build/phet-io/ph-scale_all_phet-io.html?continuousTest=%7B%22test%22%3A%5B%22ph-scale%22%2C%22fuzz%22%2C%22built-phet-io%22%5D%2C%22snapshotName%22%3A%22snapshot-1629275039864%22%2C%22timestamp%22%3A1629277464835%7D&fuzz&memoryLimit=1000&phetioStandalone
Query: fuzz&memoryLimit=1000&phetioStandalone
Uncaught TypeError: Cannot read property 'length' of undefined
TypeError: Cannot read property 'length' of undefined
at Object.numberOfDecimalPlaces (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/ph-scale/build/phet-io/ph-scale_all_phet-io.html?continuousTest=%7B%22test%22%3A%5B%22ph-scale%22%2C%22fuzz%22%2C%22built-phet-io%22%5D%2C%22snapshotName%22%3A%22snapshot-1629275039864%22%2C%22timestamp%22%3A1629277464835%7D&fuzz&memoryLimit=1000&phetioStandalone:857:95640)
at Object.roundToInterval (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/ph-scale/build/phet-io/ph-scale_all_phet-io.html?continuousTest=%7B%22test%22%3A%5B%22ph-scale%22%2C%22fuzz%22%2C%22built-phet-io%22%5D%2C%22snapshotName%22%3A%22snapshot-1629275039864%22%2C%22timestamp%22%3A1629277464835%7D&fuzz&memoryLimit=1000&phetioStandalone:857:95713)
at EV.drag [as _dragListener] (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/ph-scale/build/phet-io/ph-scale_all_phet-io.html?continuousTest=%7B%22test%22%3A%5B%22ph-scale%22%2C%22fuzz%22%2C%22built-phet-io%22%5D%2C%22snapshotName%22%3A%22snapshot-1629275039864%22%2C%22timestamp%22%3A1629277464835%7D&fuzz&memoryLimit=1000&phetioStandalone:857:1452055)
at EV.drag (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/ph-scale/build/phet-io/ph-scale_all_phet-io.html?continuousTest=%7B%22test%22%3A%5B%22ph-scale%22%2C%22fuzz%22%2C%22built-phet-io%22%5D%2C%22snapshotName%22%3A%22snapshot-1629275039864%22%2C%22timestamp%22%3A1629277464835%7D&fuzz&memoryLimit=1000&phetioStandalone:857:570812)
at _dragAction.e.Z.parameters.name (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/ph-scale/build/phet-io/ph-scale_all_phet-io.html?continuousTest=%7B%22test%22%3A%5B%22ph-scale%22%2C%22fuzz%22%2C%22built-phet-io%22%5D%2C%22snapshotName%22%3A%22snapshot-1629275039864%22%2C%22timestamp%22%3A1629277464835%7D&fuzz&memoryLimit=1000&phetioStandalone:857:960638)
at m.execute (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/ph-scale/build/phet-io/ph-scale_all_phet-io.html?continuousTest=%7B%22test%22%3A%5B%22ph-scale%22%2C%22fuzz%22%2C%22built-phet-io%22%5D%2C%22snapshotName%22%3A%22snapshot-1629275039864%22%2C%22timestamp%22%3A1629277464835%7D&fuzz&memoryLimit=1000&phetioStandalone:857:1882)
at EV.drag (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/ph-scale/build/phet-io/ph-scale_all_phet-io.html?continuousTest=%7B%22test%22%3A%5B%22ph-scale%22%2C%22fuzz%22%2C%22built-phet-io%22%5D%2C%22snapshotName%22%3A%22snapshot-1629275039864%22%2C%22timestamp%22%3A1629277464835%7D&fuzz&memoryLimit=1000&phetioStandalone:857:961674)
at EV.pointerMove (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/ph-scale/build/phet-io/ph-scale_all_phet-io.html?continuousTest=%7B%22test%22%3A%5B%22ph-scale%22%2C%22fuzz%22%2C%22built-phet-io%22%5D%2C%22snapshotName%22%3A%22snapshot-1629275039864%22%2C%22timestamp%22%3A1629277464835%7D&fuzz&memoryLimit=1000&phetioStandalone:857:573268)
at To.dispatchToListeners (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/ph-scale/build/phet-io/ph-scale_all_phet-io.html?continuousTest=%7B%22test%22%3A%5B%22ph-scale%22%2C%22fuzz%22%2C%22built-phet-io%22%5D%2C%22snapshotName%22%3A%22snapshot-1629275039864%22%2C%22timestamp%22%3A1629277464835%7D&fuzz&memoryLimit=1000&phetioStandalone:857:661502)
at To.dispatchEvent (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/ph-scale/build/phet-io/ph-scale_all_phet-io.html?continuousTest=%7B%22test%22%3A%5B%22ph-scale%22%2C%22fuzz%22%2C%22built-phet-io%22%5D%2C%22snapshotName%22%3A%22snapshot-1629275039864%22%2C%22timestamp%22%3A1629277464835%7D&fuzz&memoryLimit=1000&phetioStandalone:857:661132)
id: Bayes Chrome
Snapshot from 8/18/2021, 2:23:59 AM

----------------------------------

ph-scale : fuzz : unbuilt
https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/ph-scale/ph-scale_en.html?continuousTest=%7B%22test%22%3A%5B%22ph-scale%22%2C%22fuzz%22%2C%22unbuilt%22%5D%2C%22snapshotName%22%3A%22snapshot-1629275039864%22%2C%22timestamp%22%3A1629276217123%7D&brand=phet&ea&fuzz&memoryLimit=1000
Query: brand=phet&ea&fuzz&memoryLimit=1000
Uncaught Error: Assertion failed: scientific notation is not supported: 1e-15
Error: Assertion failed: scientific notation is not supported: 1e-15
at window.assertions.assertFunction (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/assert/js/assert.js:25:13)
at Object.numberOfDecimalPlaces (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/dot/js/Utils.js:835:17)
at Object.roundToInterval (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/dot/js/Utils.js:853:13)
at GraphIndicatorDragListener.drag [as _dragListener] (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/ph-scale/js/common/view/graph/GraphIndicatorDragListener.js:64:37)
at GraphIndicatorDragListener.drag (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/scenery/js/listeners/PressListener.js:477:10)
at DragListener._dragAction.Action.parameters.name (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/scenery/js/listeners/DragListener.js:251:36)
at Action.execute (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/axon/js/Action.js:227:18)
at GraphIndicatorDragListener.drag (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/scenery/js/listeners/DragListener.js:407:22)
at GraphIndicatorDragListener.pointerMove (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/scenery/js/listeners/PressListener.js:849:14)
at Input.dispatchToListeners (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/scenery/js/input/Input.js:1885:25)
id: Bayes Chrome
Snapshot from 8/18/2021, 2:23:59 AM

----------------------------------

ph-scale : multitouch-fuzz : unbuilt
https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/ph-scale/ph-scale_en.html?continuousTest=%7B%22test%22%3A%5B%22ph-scale%22%2C%22multitouch-fuzz%22%2C%22unbuilt%22%5D%2C%22snapshotName%22%3A%22snapshot-1629275039864%22%2C%22timestamp%22%3A1629276565595%7D&brand=phet&ea&fuzz&fuzzPointers=2&memoryLimit=1000&supportsPanAndZoom=false
Query: brand=phet&ea&fuzz&fuzzPointers=2&memoryLimit=1000&supportsPanAndZoom=false
Uncaught Error: Assertion failed: scientific notation is not supported: 1e-8
Error: Assertion failed: scientific notation is not supported: 1e-8
at window.assertions.assertFunction (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/assert/js/assert.js:25:13)
at Object.numberOfDecimalPlaces (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/dot/js/Utils.js:835:17)
at Object.roundToInterval (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/dot/js/Utils.js:853:13)
at GraphIndicatorDragListener.drag [as _dragListener] (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/ph-scale/js/common/view/graph/GraphIndicatorDragListener.js:64:37)
at GraphIndicatorDragListener.drag (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/scenery/js/listeners/PressListener.js:477:10)
at DragListener._dragAction.Action.parameters.name (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/scenery/js/listeners/DragListener.js:251:36)
at Action.execute (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/axon/js/Action.js:227:18)
at GraphIndicatorDragListener.drag (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/scenery/js/listeners/DragListener.js:407:22)
at GraphIndicatorDragListener.pointerMove (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/scenery/js/listeners/PressListener.js:849:14)
at Input.dispatchToListeners (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/scenery/js/input/Input.js:1885:25)
id: Bayes Chrome
Snapshot from 8/18/2021, 2:23:59 AM

----------------------------------

ph-scale : pan-and-zoom-fuzz : unbuilt
https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/ph-scale/ph-scale_en.html?continuousTest=%7B%22test%22%3A%5B%22ph-scale%22%2C%22pan-and-zoom-fuzz%22%2C%22unbuilt%22%5D%2C%22snapshotName%22%3A%22snapshot-1629275039864%22%2C%22timestamp%22%3A1629277694420%7D&brand=phet&ea&fuzz&fuzzPointers=2&memoryLimit=1000&supportsPanAndZoom=true
Query: brand=phet&ea&fuzz&fuzzPointers=2&memoryLimit=1000&supportsPanAndZoom=true
Uncaught Error: Assertion failed: scientific notation is not supported: 1e-12
Error: Assertion failed: scientific notation is not supported: 1e-12
at window.assertions.assertFunction (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/assert/js/assert.js:25:13)
at Object.numberOfDecimalPlaces (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/dot/js/Utils.js:835:17)
at Object.roundToInterval (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/dot/js/Utils.js:853:13)
at GraphIndicatorDragListener.drag [as _dragListener] (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/ph-scale/js/common/view/graph/GraphIndicatorDragListener.js:64:37)
at GraphIndicatorDragListener.drag (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/scenery/js/listeners/PressListener.js:477:10)
at DragListener._dragAction.Action.parameters.name (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/scenery/js/listeners/DragListener.js:251:36)
at Action.execute (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/axon/js/Action.js:227:18)
at GraphIndicatorDragListener.drag (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/scenery/js/listeners/DragListener.js:407:22)
at GraphIndicatorDragListener.pointerMove (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/scenery/js/listeners/PressListener.js:849:14)
at Input.dispatchToListeners (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/scenery/js/input/Input.js:1885:25)
id: Bayes Chrome
Snapshot from 8/18/2021, 2:23:59 AM

----------------------------------

ph-scale : phet-io-fuzz : unbuilt
https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/ph-scale/ph-scale_en.html?continuousTest=%7B%22test%22%3A%5B%22ph-scale%22%2C%22phet-io-fuzz%22%2C%22unbuilt%22%5D%2C%22snapshotName%22%3A%22snapshot-1629275039864%22%2C%22timestamp%22%3A1629276063389%7D&ea&brand=phet-io&phetioStandalone&fuzz&memoryLimit=1000
Query: ea&brand=phet-io&phetioStandalone&fuzz&memoryLimit=1000
Uncaught Error: Assertion failed: scientific notation is not supported: 1e-10
Error: Assertion failed: scientific notation is not supported: 1e-10
at window.assertions.assertFunction (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/assert/js/assert.js:25:13)
at Object.numberOfDecimalPlaces (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/dot/js/Utils.js:835:17)
at Object.roundToInterval (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/dot/js/Utils.js:853:13)
at GraphIndicatorDragListener.drag [as _dragListener] (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/ph-scale/js/common/view/graph/GraphIndicatorDragListener.js:64:37)
at GraphIndicatorDragListener.drag (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/scenery/js/listeners/PressListener.js:477:10)
at DragListener._dragAction.Action.parameters.name (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/scenery/js/listeners/DragListener.js:251:36)
at Action.execute (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/axon/js/Action.js:227:18)
at GraphIndicatorDragListener.drag (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/scenery/js/listeners/DragListener.js:407:22)
at GraphIndicatorDragListener.pointerMove (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/scenery/js/listeners/PressListener.js:849:14)
at Input.dispatchToListeners (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/scenery/js/input/Input.js:1885:25)
id: Bayes Chrome
Snapshot from 8/18/2021, 2:23:59 AM

----------------------------------

ph-scale : phet-io-mirror-inputs-fuzz : unbuilt
https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/phet-io-wrappers/mirror-inputs/?sim=ph-scale&phetioDebug&fuzz&wrapperContinuousTest=%7B%22test%22%3A%5B%22ph-scale%22%2C%22phet-io-mirror-inputs-fuzz%22%2C%22unbuilt%22%5D%2C%22snapshotName%22%3A%22snapshot-1629275039864%22%2C%22timestamp%22%3A1629277061050%7D
Uncaught Error: Uncaught Error: Assertion failed: scientific notation is not supported: 1e-11
Error: Assertion failed: scientific notation is not supported: 1e-11
at window.assertions.assertFunction (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/assert/js/assert.js:25:13)
at Object.numberOfDecimalPlaces (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/dot/js/Utils.js:835:17)
at Object.roundToInterval (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/dot/js/Utils.js:853:13)
at GraphIndicatorDragListener.drag [as _dragListener] (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/ph-scale/js/common/view/graph/GraphIndicatorDragListener.js:64:37)
at GraphIndicatorDragListener.drag (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/scenery/js/listeners/PressListener.js:477:10)
at DragListener._dragAction.Action.parameters.name (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/scenery/js/listeners/DragListener.js:251:36)
at Action.execute (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/axon/js/Action.js:227:18)
at GraphIndicatorDragListener.drag (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/scenery/js/listeners/DragListener.js:407:22)
at GraphIndicatorDragListener.pointerMove (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/scenery/js/listeners/PressListener.js:849:14)
at Input.dispatchToListeners (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/scenery/js/input/Input.js:1885:25)
id: Bayes Chrome
Snapshot from 8/18/2021, 2:23:59 AM

----------------------------------

ph-scale : phet-io-studio-fuzz : unbuilt
https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/studio/?sim=ph-scale&phetioDebug&fuzz&wrapperContinuousTest=%7B%22test%22%3A%5B%22ph-scale%22%2C%22phet-io-studio-fuzz%22%2C%22unbuilt%22%5D%2C%22snapshotName%22%3A%22snapshot-1629275039864%22%2C%22timestamp%22%3A1629276993379%7D
Uncaught Error: Uncaught Error: Assertion failed: scientific notation is not supported: 1e-13
Error: Assertion failed: scientific notation is not supported: 1e-13
at window.assertions.assertFunction (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/assert/js/assert.js:25:13)
at Object.numberOfDecimalPlaces (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/dot/js/Utils.js:835:17)
at Object.roundToInterval (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/dot/js/Utils.js:853:13)
at GraphIndicatorDragListener.drag [as _dragListener] (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/ph-scale/js/common/view/graph/GraphIndicatorDragListener.js:64:37)
at GraphIndicatorDragListener.drag (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/scenery/js/listeners/PressListener.js:477:10)
at DragListener._dragAction.Action.parameters.name (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/scenery/js/listeners/DragListener.js:251:36)
at Action.execute (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/axon/js/Action.js:227:18)
at GraphIndicatorDragListener.drag (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/scenery/js/listeners/DragListener.js:407:22)
at GraphIndicatorDragListener.pointerMove (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/scenery/js/listeners/PressListener.js:849:14)
at Input.dispatchToListeners (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/scenery/js/input/Input.js:1885:25)
id: Bayes Chrome
Snapshot from 8/18/2021, 2:23:59 AM

----------------------------------

ph-scale : xss-fuzz
https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/ph-scale/ph-scale_en.html?continuousTest=%7B%22test%22%3A%5B%22ph-scale%22%2C%22xss-fuzz%22%5D%2C%22snapshotName%22%3A%22snapshot-1629275039864%22%2C%22timestamp%22%3A1629276475738%7D&brand=phet&ea&fuzz&stringTest=xss&memoryLimit=1000
Query: brand=phet&ea&fuzz&stringTest=xss&memoryLimit=1000
Uncaught Error: Assertion failed: scientific notation is not supported: 1e-13
Error: Assertion failed: scientific notation is not supported: 1e-13
at window.assertions.assertFunction (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/assert/js/assert.js:25:13)
at Object.numberOfDecimalPlaces (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/dot/js/Utils.js:835:17)
at Object.roundToInterval (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/dot/js/Utils.js:853:13)
at GraphIndicatorDragListener.drag [as _dragListener] (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/ph-scale/js/common/view/graph/GraphIndicatorDragListener.js:64:37)
at GraphIndicatorDragListener.drag (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/scenery/js/listeners/PressListener.js:477:10)
at DragListener._dragAction.Action.parameters.name (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/scenery/js/listeners/DragListener.js:251:36)
at Action.execute (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/axon/js/Action.js:227:18)
at GraphIndicatorDragListener.drag (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/scenery/js/listeners/DragListener.js:407:22)
at GraphIndicatorDragListener.pointerMove (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/scenery/js/listeners/PressListener.js:849:14)
at Input.dispatchToListeners (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1629275039864/scenery/js/input/Input.js:1885:25)
id: Bayes Chrome
Snapshot from 8/18/2021, 2:23:59 AM
```

**Comment by pixelzoom** [comment]
This is due to changes made to `Utils.numberOfDecimalPlaces` in https://github.com/phetsims/dot/issues/109.   I won't be able to investigate until 8/23 at the earliest.

**Comment by pixelzoom** [comment]
This was resolved by a fix in phetsims/dot#109. Closing.
